webpack_config/common.js

```diff
@@ -11,7 +11,7 @@ const FaviconsWebpackPlugin = require('favicons-webpack-plugin');
 const config = require('./config');
 
 const IS_DEVELOPMENT = process.env.NODE_ENV === 'development';
-const IS_ELECTRON = !!process.env.BUILD_ELECTRON;
+const IS_ELECTRON = process.env.TARGET_ENV === 'electron';
 
```

Comment:
- Use `src/utils/environment`. Or move them to a place that works for the whole project.
- Use `IS_DEV`